apps/pages/tests/test_views.py

```diff
@@ -6,7 +6,7 @@
 class PageDetailTestView(WebTest):
 
     def setUp(self):
-        self.page = PageFactory.create()
+        self.page = PageFactory.create(is_active=True)
```

Comment:
Then you probably won't need all these `is_active`!

Reply:
:man_facepalming: